odrive_ros2_control/src/odrive_hardware_interface.cpp

```diff
@@ -36,7 +36,9 @@ class ODriveHardwareInterface final : public hardware_interface::SystemInterface
 
 private:
     void on_can_msg(const can_frame& frame);
+    void set_axis_command_mode(Axis axis);
```

Comment:
`Axis axis` => `const Axis&`
(and with that the definition of `send()` needs to be changed to `const`)